src/drivers/display/renesas_glcdc/lv_renesas_glcdc.h

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file lv_renesas_glcdc.h
+ *
+ */
+
+#ifndef LV_RENESAS_GLCDC_H
+#define LV_RENESAS_GLCDC_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "../../../display/lv_display.h"
+
+#if LV_USE_RENESAS_GLCDC
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ * GLOBAL PROTOTYPES
+ **********************/
+
+/**
+ * Create a display using Renesas' GLCDC peripherial in DIRECT render mode
+ * @return  pointer to the created display
+ */
+lv_display_t * lv_renesas_glcdc_create_direct(void);
```

Comment:
Usually `create` is the last keyword in function names:
```suggestion
lv_display_t * lv_renesas_glcdc_direct_create(void);
```